eng/common/darc-init.sh

```diff
@@ -49,16 +49,15 @@ function InstallDarcCli {
   local dotnet_root=$_InitializeDotNetCli
 
   local uninstall_command=`$dotnet_root/dotnet tool uninstall $darc_cli_package_name -g`
-  local tool_list=$($dotnet_root/dotnet tool list -g)
+  local tool_list=$($dotnet_root/dotnet tool list --tool-path $dotnet_root/tools)
```

Comment:
Did you open a PR for this in Arcade?  We'll be getting auto-update PRs from it until they match.

Reply:
[#4152](https://github.com/dotnet/arcade/pull/4152) for the update